Update metrics-cloudwatch.adoc

vnandwana opened the pull request:
Added a section "Creating Alarms in CloudWatch"

vnandwana committed:
Update metrics-cloudwatch.adoc

Added a section "Creating Alarms in CloudWatch"

jdmooney reviewed (approved):
Everything looks good to me1

vnandwana commented:
@reshmabidikar This PR addresses the comment you added in the ticket -- https://github.com/killbill/aws-marketplace/issues/5#issuecomment-1529374373. Kindly look into this when you get a chance.

reshmabidikar reviewed (approved):
Looks good!
